synapse/rest/media/v1/preview_url_resource.py

```diff
@@ -144,6 +101,7 @@ def __init__(
         self.clock = hs.get_clock()
         self.filepaths = media_repo.filepaths
         self.max_spider_size = hs.config.max_spider_size
+        self.oembed = hs.config.oembed
```

Comment:
Please can you give this a more descriptive name?

```suggestion
        self.oembed_config = hs.config.oembed
```